sass/components/tag-components.scss

```
@@ -0,0 +1,75 @@
+body {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+}
+
+.btn-container {
+	width: 100%;
+	background-color: white;
+	margin: 50px 0px 200px 200px;
+}
+
+.tag-btn {
```

```suggestion
.tag {
```

Rename the class name everywhere.